src/components/index.js

```diff
@@ -1,57 +1,57 @@
-require('./animation');
-require('./anchored');
-require('./camera');
-require('./cursor');
-require('./geometry');
-require('./generic-tracked-controller-controls');
-require('./gltf-model');
-require('./grabbable');
-require('./hand-tracking-controls');
-require('./hand-tracking-grab-controls');
-require('./hand-controls');
-require('./hide-on-enter-ar');
-require('./hide-on-enter-vr');
-require('./hp-mixed-reality-controls');
-require('./layer');
-require('./laser-controls');
-require('./light');
-require('./line');
-require('./link');
-require('./logitech-mx-ink-controls');
-require('./look-controls');
-require('./magicleap-controls');
-require('./material');
-require('./obb-collider');
-require('./obj-model');
-require('./oculus-go-controls');
-require('./meta-touch-controls');
-require('./pico-controls');
-require('./position');
-require('./raycaster');
-require('./rotation');
-require('./scale');
-require('./shadow');
-require('./sound');
-require('./text');
-require('./tracked-controls');
-require('./visible');
-require('./valve-index-controls');
-require('./vive-controls');
-require('./vive-focus-controls');
-require('./wasd-controls');
-require('./windows-motion-controls');
+import './animation.js';
+import './anchored.js';
+import './camera.js';
+import './cursor.js';
+import './geometry.js';
+import './generic-tracked-controller-controls.js';
+import './gltf-model.js';
+import './grabbable.js';
+import './hand-tracking-controls.js';
+import './hand-tracking-grab-controls.js';
+import './hand-controls.js';
+import './hide-on-enter-ar.js';
+import './hide-on-enter-vr.js';
+import './hp-mixed-reality-controls.js';
+import './layer.js';
+import './laser-controls.js';
+import './light.js';
+import './line.js';
+import './link.js';
+import './logitech-mx-ink-controls.js';
+import './look-controls.js';
+import './magicleap-controls.js';
+import './material.js';
+import './meta-touch-controls.js';
+import './obb-collider.js';
+import './obj-model.js';
+import './oculus-go-controls.js';
```

Comment:
Diff looks a tad strange, as I moved `meta-touch-controls.js` up to maintain alphabetical order.